wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -22,17 +23,14 @@ void withTimeoutTest() {
     HAL.initialize(500, 0);
     SimHooks.pauseTiming();
     try (CommandScheduler scheduler = new CommandScheduler()) {
-      Command command1 = new WaitCommand(10);
-
-      Command timeout = command1.withTimeout(2);
+      Command timeout = new RunCommand(() -> {}).withTimeout(0.1);
 
       scheduler.schedule(timeout);
       scheduler.run();
 
-      assertFalse(scheduler.isScheduled(command1));
```

Comment:
Why drop this assertion?

Reply:
It felt like something that would be caught in composition semantics, but I suppose there's no reason to remove it.

Reply:
Then again, C++ compositions own their commands, so I really can't check if the command is scheduled when I have to move it in order to use the decorator.